src/index.js

```diff
@@ -10,7 +10,7 @@ form.addEventListener('submit', (e) => {
 
```

Comment:
![License-link](https://user-images.githubusercontent.com/93039359/196643697-ba656367-bc78-4e24-9171-16ff523a242a.PNG)

- [ ] Your link to your license file is broken. Kindly fix it 👍